### PR TITLE
Fixed qunit test with PhantomJS

### PR DESCRIPTION
**Pull request by ar4s**
Now it works!

### CODE REVIEW

**Comment by szok** [comment]
:ok:

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +14,7 @@ before_install:
     - make flake
 
 install:
+    - npm config set python /usr/bin/python2.7
```
Comment (mkurek) [inline .travis.yml:17]: why can't we use python3?

Comment (ar4s) [inline .travis.yml:17]: because some dependency (`node-gyp`) work only with Python 2.x., JS's tests and Python's tests are separeted from each other

**Comment by mkurek** [comment]
:ok:

**Pull request merged**